lib/modules/commentNavigator.js

```diff
@@ -297,26 +301,19 @@ export function toggleNavigator() {
 	}
 }
 
-export function getPostsByCategory() {
-	if (currentCategory) {
-		if (!_posts[currentCategory] || currentCategory === 'highlighted') {
-			_posts[currentCategory] = sortTypes[currentCategory].getPosts();
-		}
-	}
+export function refreshCurrentCategoryPosts() {
+	_posts[currentCategory] = sortTypes[currentCategory] ? sortTypes[currentCategory].getPosts() : sortTypes[currentCategory];
```

Comment:
Oh, I forgot that this change you suggested causes the flow check to fail with really cryptic messages.